HotelManagement/src/core/Manager.java

```diff
@@ -76,15 +78,15 @@ public void bookRoom(int roomNumber) {
 	 * Returns the available rooms of the hotel.
 	 * @return ArrayList with the available rooms of the hotel
 	 */
-	public ArrayList<Room> getAvailableRooms() {
-		ArrayList<Room> rooms = this.hotel.getRooms();
-		ArrayList<Room> availableRooms = new ArrayList<>();
+	public void findAvailableDatesForIntervalAndSizeForRooms(LocalDate fromDate, LocalDate toDate, int size, int numberOfBeds) {
+		List<Room> rooms = this.hotel.getRooms();
 		for (int i = 0; i < rooms.size(); i++) {
-			if (!rooms.get(i).isBooked()) {
-				availableRooms.add(rooms.get(i));
+			if (rooms.get(i).getNumberOfBeds() >= numberOfBeds) {
+				ArrayList<String> availableDates = rooms.get(i).findAvailableDatesForIntervalAndSize(fromDate, toDate, size);
+				for (String availableDate : availableDates) {
+					System.out.println(availableDate);
```

Comment:
How can you use the available dates that you have printed out?

Reply:
The idea was to print them and customer to choose date interval, going to change it soon when the new task comes